### PR TITLE
Reduce Comment Size from codecov

### PR DESCRIPTION
**Pull request by AustinShalit**
Depends on #118 

I do not know why disabling codecov is not working right now.  This PR will reduce the size of codecov comments to make them less obtrusive.  

Please wait to merge until codecov comments on this PR so we can confirm that we like it.

### CODE REVIEW

**Comment by SamCarlberg** [comment]
Please rebase onto `master` instead of `fix117`

**Comment by AustinShalit** [comment]
I can't.  Master does not pass all CI.  codecov will not comment unless CI passes.

**Comment by JLLeitschuh** [comment]
> Please rebase onto master instead of fix117

> Depends on #118

File: codecov.yml [hunk 1]
```diff
@@ -5,11 +5,14 @@ codecov:
 coverage:
   precision: 2
   round: down
-  range: "70...100"
+  range: "60...100"
```
Comment (nightpool) [inline codecov.yml]: this change seems unrelated and also not great.

Comment (AustinShalit) [inline codecov.yml]: Changed back to 70

Comment (JLLeitschuh) [inline codecov.yml]: I think that saying that code coverage isn't important is like saying, you enjoy writing legacy code.

> Legacy code is code without tests
> \- Michael Feathers' Working Effectively with Legacy Code (ISBN 0-13-117705-2)

Theres two sides to this, you aren't writing tests for yourself, you are writing tests for the next developer that comes along and is trying to add a new feature without breaking existing functionality.
